std_msgs/Bool closed_path
geometry_msgs/Point[] points